.gitignore

```diff
@@ -4,3 +4,4 @@ aws.yml
 build/
 .powenv
 .envrc
+.idea/
```

Comment:
I usually add this to the global .gitignore file, in this case you don't need to add it in each repo you are working on